core-blocks/button/edit.js

```diff
@@ -35,8 +35,8 @@ const { getComputedStyle } = window;
 
 const FallbackStyles = withFallbackStyles( ( node, ownProps ) => {
 	const { textColor, backgroundColor } = ownProps;
-	const backgroundColorValue = backgroundColor && backgroundColor.value;
-	const textColorValue = textColor && textColor.value;
+	const backgroundColorValue = backgroundColor && backgroundColor.color;
```

Comment:
`backgroundColorValue` - we still use old const name which in my opinion indicates that we shouldn't rename property from `value` to `color`. It also seems like having color twice in a construct like `textColor.color` indicates that `value` is more accurate.

Reply:
Themes set the color object with a color property. If we want to use value should themes also pass a value property?